routing/routing.go

```diff
@@ -144,6 +144,7 @@ type RouteFilter struct {
 // backends.
 type LBEndpoint struct {
 	Scheme, Host string
+	Detected     time.Time
```

Comment:
can you please add godoc?

Reply:
sure. (You can ignore the missing godoc momentarily, the PR is just a draft currently, and there is no godoc yet anywhere.)